_includes/footer.html

```diff
@@ -2,7 +2,84 @@
 <script src="https://code.jquery.com/jquery-3.5.1.min.js"
   integrity="sha256-9/aliU8dGd2tb6OSsuzixeV4y/faTqgFtohetphbbj0=" crossorigin="anonymous"></script>
 <script type="text/javascript" src="{{ "/assets/bootstrap/js/bootstrap.min.js" | relative_url }}"></script>
+<script src="https://cdn.jsdelivr.net/npm/chart.js@3.7.1/dist/chart.min.js"></script>
 <script type="text/javascript">
+  function createChart(labels, dat) {
+    const ctx = $("#weeklyCommits");
+    const data = {
+      labels: labels,
+      datasets: [{
+        data: dat,
+        backgroundColor: "#FFAD33",

```

Comment:
I think we should have CSS "Ruffle blue" and "Ruffle orange" color constants, like we have in Ruffle itself:
https://github.com/ruffle-rs/ruffle/blob/b8cdd598771927e7229cd633599771eb2161f5bd/web/packages/core/src/shadow-template.ts#L9-L10